readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -0,0 +1,504 @@
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Enhanced full-text search</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon orange big">Ad-free</div>
+
+            <div class="ui header huge"><small><i>from</i></small> $5</div>
+            <div class="meta">
+              <span class="category">one-time donation</span>
+              <span class="right floated"><i>limited time</i></span>
+            </div>
+
+            <button class="ui button fluid large orange">Donate</button>
+
+            <div class="description">
+              
+              <div class="ui list relaxed divided">
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Automatic build</div>
+                    <div class="description">on every commit</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Multi-version handling</div>
+                    <div class="description">from branches or tags</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Downloadable formats</div>
+                    <div class="description">PDF, HTML</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Enhanced full-text search</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Ad-free user</div>
+                    <div class="description">stop seeing ads on any Read the Docs hosted documentation when logged in your account</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon yellow big">Gold</div>
+
+            <div class="ui header huge"><small><i>from</i></small> $5</div>
+            <div class="meta">
+              <span class="category">per month</span>
+            </div>
+
+            <button class="ui button fluid large yellow">Subscribe</button>
+
+            <div class="description">
+              
+              <div class="ui list relaxed divided">
```

Comment:
Should we mimic above and instead use "All open source plan features, plus:" here? The only difference is ads are removed

Reply:
This was done in commit [41dcffd](https://github.com/readthedocs/site-community/pull/34/commits/41dcffdde613f3d03775c407bfa83993c443fe66).